bash_tests/language_translator_1.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# The following will translate two phrases, "Hello, world!" and "How are you?" from English to Spanish.
+
+echo "Translate two phrases from English to Spanish"
+
+if test "$#" -ne 1; then
+    echo "Usage: $0 api-key"
+else
+	echo " Calling Watson Service API Using -> \"apikey:$1\""
+	curl -X POST -u "apikey:$1" \
+	--header "Content-Type: application/json" \
+	--data "{\"text\": [\"Hello, world! \", \"How are you?\"], \"model_id\":\"en-es\"}" \
+	"https://gateway-lon.watsonplatform.net/language-translator/api/v3/translate?version=2018-05-01"
```

Comment:
Is this gateway going to work for everyone? What if I create my service in another zone?

Reply:
Yes, this has nothing about zones. but it might be changed if a new WATSON API version is released. I will add comment to inform users to update gateway when required.